bin/rialto-parachain/node/Cargo.toml

```diff
@@ -66,15 +66,15 @@ sp-timestamp = { git = "https://github.com/paritytech/substrate", branch = "mast
 sp-transaction-pool = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
 # Cumulus dependencies
-cumulus-client-consensus-aura = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-client-consensus-common = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-client-cli = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-client-network = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-client-service = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-primitives-core = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-primitives-parachain-inherent = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-relay-chain-interface = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
-cumulus-relay-chain-inprocess-interface = { git = "https://github.com/paritytech/cumulus", branch = "gav-xcm-v3" }
+cumulus-client-consensus-aura = { git = "https://github.com/paritytech/cumulus", branch = "master" }
```

Comment:
So there isn't any xcmv3 specific stuff in `cumulus`?

Reply:
we need to have the same version of xcm/substrate/polkadot dependencies everywhere,
because bridges repo are based on origin/gav-xcm-v3 + substrate-at-some-commit

Reply:
I'm also not sure why you've changed from `gav-xcm-v3` to `master` here :) But if it works for you and for Rialto/Millau, then it is fine :)

Reply:
@svyatonik 
well, I just wanted to align it with **polkadot's** `locked-for-gav-xcm-v3-and-bridges`  and **substrate's** `sv-locked-for-gav-xcm-v3-and-bridges`, everywhere we use `branch=master` and patched branch version is just in main Cargo.toml,
so I just wanted to do the same for Cumulus repo deps

Reply:
ahhh, ok - I see :) thank you for the explanation! :)